docs/Configuration.md

```diff
@@ -909,9 +909,7 @@ If you need to restrict your test-runner to only run in serial rather than being
 
 Default: `[]`
 
-A list of paths to modules that run some code to configure or set up the testing environment. Each setupFile will be run once per test file. Since every test runs in its own environment, these scripts will be executed in the testing environment immediately before executing the test code itself.
-
-It's also worth noting that `setupFiles` will execute _before_ [`setupFilesAfterEnv`](#setupfilesafterenv-array).
+A list of paths to modules that run some code to configure or set up the testing environment. Each setupFile will be run once per test file. Since every test runs in its own environment, these scripts will be executed in the testing environment before executing [`setupFilesAfterEnv`](#setupfilesafterenv-array) and before the test code itself.
```

Comment:
from below it

>this script file presents you the opportunity of running some code immediately after the test framework has been installed in the environment.

Should we state the opposite here, somehow? That it executes before the test framework has been installed?

Reply:
This bit seems clear enough to me, but for completeness I added clause to say that it executes before the tests.